webpack.config.js

```diff
@@ -63,6 +63,11 @@ function getBundleAnalyzerPlugin(analyzeBundle, name) {
  * target, undefined; otherwise, the path to the local file to be served.
  */
 function devServerProxyBypass({ path }) {
+    if (path.startsWith('/v1/_cdn/')) {
+        // The CDN is not available in the dev server, so we need to bypass it.
+        path = path.replace(/\/v1\/_cdn\/[^/]+\//, '/');
```

Comment:
```suggestion
        return path.replace(/\/v1\/_cdn\/[^/]+\//, '/');
```

Reply:
Ah wait, not quite. This will correctly format the URL, but it will be absolute, not relative, so the following checks will fail, I think.

We'll also need to cleanup the query parameters, so the matching below works.